vision/cloud-client/detect/detect.py

```diff
@@ -636,6 +641,77 @@ def detect_document_uri(uri):
 # [END def_detect_document_uri]
 
 
+# [START vision_async_detect_document_ocr]
+def async_detect_document(gcs_source_uri, gcs_destination_uri):
+    """OCR with PDF/TIFF as source files on GCS"""
+    # Supported mime_types are: 'application/pdf' and 'image/tiff'
+    mime_type = 'application/pdf'
+
+    # How many pages should be grouped into each json output file.
+    # With a file of 5 pages
```

Comment:
I don't understand this part, "with a file of 5 pages", is it cut off?

Reply:
The API lets you decide how many pages you want each output file to be. 

So its reads a PDF and it writes a JSON file. Then you can decide each json file gets 1-### of pages of the PDF.

Reply:
That makes sense, but what does the sentence fragment "With a file of 5 pages" mean specifically?

Reply:
Oh... Yea, I think that is referring to the sample PDF we provide, I'll pull that.